cmd/runtimetest/main.go

```diff
@@ -257,9 +257,12 @@ func main() {
 		validateROPaths,
 	}
 
+	ret := 0
+	defer os.Exit(ret)
```

Comment:
Why defer here, instead of calling `os.Exit` after the loop completes?  The only case I can see is if validation panics, and presumably that is already leading to a non-zero exit status.